crates/fluvio-package-index/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-package-index"
-version = "0.6.1"
+version = "0.6.2"
```

Comment:
Since we are changing API, the version should be bumped to `0.7.0`

Reply:
Done. `fluvio-extension-common` depends on this crate. Should I bump the patch or minor version on that case?

Reply:
Looks like `fluvio-extension-common` only needs a small API from `fluvio-package-index` and shouldn't bring the entire HTTP stack.   Let's add a feature flag to `fluvio-extension-common` to make package retrieval optional.

Reply:
Done